internal/configs/sharding/localnet.go

```diff
@@ -14,8 +14,12 @@ import (
 // configuration schedule.
 var LocalnetSchedule localnetSchedule
 
-// privatekey: 0x1111111111111111111111111111111111111111111111111111111111111111
-var feeCollectorLocalnet = mustAddress("0x19E7E376E7C213B7E7e7e46cc70A5dD086DAff2A")
+var feeCollectorsLocalnet = FeeCollectors{
+	// 0x1111111111111111111111111111111111111111111111111111111111111111
+	mustAddress("0x19E7E376E7C213B7E7e7e46cc70A5dD086DAff2A"): numeric.MustNewDecFromStr("0.5"),
+	// 0x2222222222222222222222222222222222222222222222222222222222222222
+	mustAddress("0x1563915e194D8CfBA1943570603F7606A3115508"): numeric.MustNewDecFromStr("0.5"),
+}
```

Comment:
are the private key or mnemonic of the collector in localnet somewhere available in the code ?